python/lsst/ts/MTAlignment/alignment_csc.py

```diff
@@ -144,191 +163,356 @@ async def configure(self, config):
-        T2SA host."""
+    async def do_loadSATemplateFile(self, data: salobj.BaseDdsDataType) -> None:
+        """Load SA Template file.
+
+        Parameters
+        ----------
+        data : ``cmd_loadSATemplateFile.DataType``
+            Command data.
+        """
         self.assert_enabled()
+        assert self.model is not None
         await self.model.load_template_file(data.file)
 
-    async def do_measureDrift(self, data):
-        """measure tracker drift"""
+        # TODO: Publish something?
+
+    async def do_measureDrift(self, data: salobj.BaseDdsDataType) -> None:
+        """Measure tracker drift.
+
+        Parameters
+        ----------
+        data : ``cmd_measureDrift.DataType``
+            Command data.
+        """
         self.assert_enabled()
+        assert self.model is not None
         await self.model.measure_drift(data.pointgroup)
 
-    async def do_resetT2SA(self, data):
-        """reboots t2sa and SA"""
+        # TODO: Publish something?
+
+    async def do_resetT2SA(self, data: salobj.BaseDdsDataType) -> None:
+        """Reboots t2sa and SA.
+
+        Parameters
+        ----------
+        data : ``cmd_resetT2SA.DataType``
+            Command data.
+        """
         self.assert_enabled()
+        assert self.model is not None
+
         await self.model.reset_t2sa()
 
-    async def do_newStation(self, data):
-        """create new tracker station"""
+        # TODO: Publish something?
+
+    async def do_newStation(self, data: salobj.BaseDdsDataType) -> None:
+        """Add new tracker station.
+
+        Parameters
+        ----------
+        data : ``cmd_newStation.DataType``
+            Command data.
+        """
         self.assert_enabled()
-        await self.model.new_station(data.newStation)
+        assert self.model is not None
+
+        await self.model.new_station()
+
+        # TODO: Publish something?
 
-    async def do_saveJobfile(self, data):
-        """save job file"""
+    async def do_saveJobfile(self, data: salobj.BaseDdsDataType) -> None:
+        """Save job file.
+
+        Parameters
+        ----------
+        data : ``cmd_saveJobFile.DataType``
+            Command data.
+        """
         self.assert_enabled()
+        assert self.model is not None
+
         await self.model.save_sa_jobfile(data.file)
 
-    async def correction_loop(self):
+        # TODO: Publish something?
+
+    async def correction_loop(self, target: str) -> None:
+        """Measure the offset between M1M3 optimum position and the target.
+
+        Parameters
+        ----------
+        target : `str`
+            Target to measure offset with respect to M1M3 optimum position.
+        """
+
+        assert self.model is not None
+
         await self.model.set_telescope_position(
-            self.elevation, self.azimuth, self.camrot
+            telalt=self.elevation,
+            telaz=self.azimuth,
+            camrot=self.camrot,
```

Comment:
As far as I understand, this are mostly used by T2SA to create collections of data. I think it comes from a misunderstanding about how we will use this system. Because the requirement is for the telescope to be at a specific position they thought we might have to pass this information to the T2SA controller, which, in truth, is not required.

Reply:
I suggest talking to Scott Sandwith about this. If the T2SA isn't doing anything useful with the information, then ditch this command.

Note that Scott is doing some work on T2SA right now, so this would be a good time to bring this up.

Reply:
I agree.. Who's the right person to spearhead this? I am not in contact with Scott. Maybe you or Dave?

Reply:
At this point I think @dmills-lsst. If it can wait until November or so I can also help.